src/templates/PoolManager/ActivityCard/TokenChangeUpdate/styles.ts

```diff
@@ -0,0 +1,53 @@
+import styled, { css } from 'styled-components'
+
+export const TokenChangeUpdate = styled.div`
+  ${() => css`
+    display: flex;
+    justify-content: space-between;
+    gap: 2.4rem;
+
+    @media (max-width: 600px) {
+      flex-direction: column;
+      padding-left: 2.4rem;
+    }
+  `}
+`
```

Comment:
You're not using any variable from `styled-components`, this arrow function is unnecessary
```suggestion
export const TokenChangeUpdate = styled.div`
  display: flex;
  justify-content: space-between;
  gap: 2.4rem;

  @media (max-width: 600px) {
    flex-direction: column;
    padding-left: 2.4rem;
  }
`
```